tests/end-to-end/upgrade/pkg/tests/monitoring/targets_rules.go

```diff
@@ -204,11 +203,33 @@ func shouldIgnoreTarget(target prom.Labels) bool {
 		"knative-eventing/knative-eventing-event-mesh-dashboard-httpsource",
 	}
 
+	var podsToBeIgnored = []string{
+		// Ignore the pods that are created during tests.
+		"-testsuite-",
+		"test",
+		"nodejs12-",
+		"nodejs10-",
+		"upgrade",
+	}
+
+	var namespacesToIgnore = "test"
+
+	for _, p := range podsToBeIgnored {
+		if strings.Contains(target["pod_name"], p) {
+			return true
+		}
+	}
+
 	for _, j := range jobsToBeIgnored {
 		if target["job"] == j {
 			return true
 		}
 	}
+
+	if strings.Contains(target["namespace"], namespacesToIgnore) {
```

Comment:
Shouldn't this be either `strings.EqualFold` (for case insensitive) or just `target["namespace"] == namespacesToIgnore`?

The current code would return true in the following scenario, which probably is not correct:
```go
target["namespace"] = "testsFromCustomer"
namespacesToIgnore = "test"
```